integreat_cms/cms/models/contact/contact.py

```diff
@@ -91,6 +92,21 @@ def get_additional_attribute(self) -> str:
 
         return ""
 
+    def label_in_reference_list(self) -> str:
+        """
+        This function returns
+        """
+
+        label = (
+            _("General contact information")
+            if not self.point_of_contact_for
+            else f"{self.point_of_contact_for} {self.name}"
+        )
+        if self.archived:
+            label += " (⚠ " + gettext("Archived") + ")"
```

Comment:
I don't think that it makes a difference here which translation function is used, so imo it would be nicer to not import `gettext_lazy`

Probably a nit though :)

Reply:
What I tried it doesn't make a difference in the end. It's probably more consistent with our documentation with `gettext_lazy` and since it's already imported, it would spare one import. But it's not really that important